loader/windows/icd_windows_dxgk.c

```diff
@@ -83,47 +88,47 @@ bool khrIcdOsVendorsEnumerateDXGK(void)
             KHR_ICD_TRACE("D3DKMT_ENUMADAPTERS2 status != SUCCESS\n");
             goto out;
         }
+        const char* cszOpenCLRegKeyName = GetOpenCLRegKeyName();
+        const int szOpenCLRegKeyName = (int)(strlen(cszOpenCLRegKeyName) + 1)*sizeof(cszOpenCLRegKeyName[0]);
```

Comment:
the 1 can be replaced with sizeof(cszOpenCLRegKeyName[0])) for (possible) future compatibility with unicode.

Reply:
Shouldn't this be taken care by outer sizeof(cszOpenCLRegKeyName[0])? Multiplication by sizeof(cszOpenCLRegKeyName[0]) applies to 1 as well.